src/tsc/tsc.ts

```diff
@@ -133,6 +133,11 @@ namespace ts {
         if (configFileName) {
             const configParseResult = parseConfigFileWithSystem(configFileName, commandLineOptions, sys, reportDiagnostic)!; // TODO: GH#18217
             if (commandLineOptions.showConfig) {
+                if (configParseResult.errors.length !== 0) {
+                    updateReportDiagnostic(configParseResult.options);
+                    configParseResult.errors.forEach(reportDiagnostic);
+                    return sys.exit(ExitStatus.DiagnosticsPresent_OutputsGenerated);

```

Comment:
I think return code should be `DiagnosticsPresent_OutputsSkipped`

Reply:
Seems reasonable.

Reply:
wow, I forgot to fix it. thanks!